framework/src/wifi_manager/wifi_manager.c

```diff
@@ -1271,8 +1286,10 @@ void _handle_user_cb(_wifimgr_usr_cb_type_e evt, void *arg)
 	default:
 		WIFIADD_ERR_RECORD(ERR_WIFIMGR_INVALID_EVENT);
 		ndbg("[WM] Invalid State\n");
+		return ;
```

Comment:
remove space before ";"
you can remove break; also.

Reply:
That is right. Modification completed. Thanks.